src/pages/store/index.js

```diff
@@ -0,0 +1,31 @@
+import Link from 'next/link'
+
+function Store () {
+  return (
+    <div className="page-content">
+      <div className="wrapper">
+        {'Thank you for your interest in supporting our mission!'}
+        <br />
+        {'Unfortunately, due to our payment provider\'s integration with our storefront no longer being supported, our store is currently offline. '}
+        {'As a result, in order to reopen our store, we have to drop our old solution and come up with a new one from the ground up. '}
+        {'This is on the proverbial to-do list, but is not a small task - as of now, we can\'t promise when it will be finished.'}
+        <br />
+        {'If you are interested in supporting us in the meantime, we greatly appreciate '}
+        <Link href="/donate">{'donations'}</Link>{'.'}
+      </div>
+    </div>
+  )
+}
+
+Store.getPageMeta = () => {
+  return {
+    title: 'Merch Store',
+    description: 'Support the Fuel Rats\' mission to aid spaceship pilots in our vast universe! Your generous donation helps us continue our in-game rescue services, ensuring no player is left stranded in space. Contribute today and fuel our journey to make the gaming experience enjoyable for everyone!',
```

Comment:
in interest of maintaining tech debt, could we at least resolve this line length issue? the others we can handle eventually, or you could split them in this PR if you'd like, but at the very least lets not introduce more work

Reply:
I have no issue with that, I just copied the meta description from the donation page. Good practice would be to have meta descriptions be ~105 chars in length max anyway.